[2024-01-29T00:34:44.951+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cron_v0.task1 manual__2024-01-28T19:04:37.580462+00:00 [queued]>
[2024-01-29T00:34:44.959+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cron_v0.task1 manual__2024-01-28T19:04:37.580462+00:00 [queued]>
[2024-01-29T00:34:44.960+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-29T00:34:44.987+0530] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): task1> on 2024-01-28 19:04:37.580462+00:00
[2024-01-29T00:34:44.989+0530] {standard_task_runner.py:60} INFO - Started process 4271 to run task
[2024-01-29T00:34:44.992+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_cron_v0', 'task1', 'manual__2024-01-28T19:04:37.580462+00:00', '--job-id', '331', '--raw', '--subdir', 'DAGS_FOLDER/5-create-dag-with-cron-exp.py', '--cfg-path', '/tmp/tmpql0oeph5']
[2024-01-29T00:34:44.996+0530] {standard_task_runner.py:88} INFO - Job 331: Subtask task1
[2024-01-29T00:34:45.061+0530] {task_command.py:423} INFO - Running <TaskInstance: dag_with_cron_v0.task1 manual__2024-01-28T19:04:37.580462+00:00 [running]> on host mh1.myguest.virtualbox.org
[2024-01-29T00:34:45.170+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='hdu' AIRFLOW_CTX_DAG_ID='dag_with_cron_v0' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T19:04:37.580462+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-28T19:04:37.580462+00:00'
[2024-01-29T00:34:45.171+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-29T00:34:45.172+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo testing cron expression']
[2024-01-29T00:34:45.182+0530] {subprocess.py:86} INFO - Output:
[2024-01-29T00:34:45.183+0530] {subprocess.py:93} INFO - testing cron expression
[2024-01-29T00:34:45.183+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-29T00:34:45.206+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_cron_v0, task_id=task1, execution_date=20240128T190437, start_date=20240128T190444, end_date=20240128T190445
[2024-01-29T00:34:45.252+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-29T00:34:45.272+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
